runner.py

```diff
@@ -682,6 +692,12 @@ def setup_services(self):
                 for network in service['networks']:
```

Comment:
You could remove the `if`? Why have 2 cases here? Does it matter if the network is specified in the usage or added by us? And you could look over `self.__networks.append(network)` and not the `services?

Reply:
it is important to only join the default network if the container has no own defined networks as one might want to separate containers.
If I look over `self.__networks.append(network) ` I would append it to any network known. This is most likely not what the user wants.